homeassistant/components/spotify/media_player.py

```diff
@@ -134,6 +134,7 @@ def __init__(
             SPOTIFY_SCOPES
         )
         self._currently_playing: dict | None = {}
+        self._currently_playing_updated: int
```

Comment:
Shouldn't it be datetime?

Reply:
And could it be None?

Reply:
Ah, yes you're correct 🤦‍♂️
I had it as a timestamp earlier but forgot to update this after changing it.

And it could be None as well yes.

I'll update shortly. Thanks!